### PR TITLE
git: add Spanish translation

### PR DESCRIPTION
**Pull request by jreyes33**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 2.34.1

This is my first translation to Spanish (and overall contribution to this project). Feedback is much appreciated, as I intend to continue collaborating with more translations.

Thank you.

### CODE REVIEW

File: pages.es/common/git.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+`git {{subcomando}}`
+
+- Ejecuta un subcomando de Git en un repositorio en la ruta raíz especificada:
+
+`git -C {{ruta/al/repositorio}} {{subcomando}}`
+
+- Ejecuta un subcomando de Git con la configuración definida:
+
+`git -c '{{clave.de.configuración}}={{valor}}' {{subcomando}}`
```
Comment (navarroaxel) [inline pages.es/common/git.md:29]: I think Git only supports 1 section and 1 key for configurations, maybe this is more similar to a real configuration value:
```suggestion
`git -c '{{config.clave}}={{valor}}' {{subcomando}}`
```

**Comment by jreyes33** [comment]
Thanks for the review, @navarroaxel! For future PRs, would it be more convenient to open a PR with multiple translations (5 or so) or do you prefer to keep a single translation per PR?

**Comment by navarroaxel** [comment]
> Thanks for the review, @navarroaxel! For future PRs, would it be more convenient to open a PR with multiple translations (5 or so) or do you prefer to keep a single translation per PR?

@jreyes33 thanks! For translations you can send more than 1 translated page in 1 PR, just keep 1 commit per file and I recommend no more than 15 pages. This doesn't apply to the main «English» pages.

**Review by navarroaxel — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Comment by navarroaxel** [comment]
> Hey @navarroaxel, was the agreement signed here? I was trying to merge it however I couldn't because the CI was reporting that it wasn't signed.

@CleanMachine1 Yeah, I saw the :heavy_check_mark: earlier, but sometimes the check is broken. Also, @jreyes33 authored #7724 and the check was green and the PR was merged by you.

**Comment by jreyes33** [comment]
FWIW, I've signed the agreement and when I opened the PR that check passed. However, after applying the suggested changes, that check was yellow all the time.

**Comment by CleanMachine1** [comment]
All fine then, just making sure, thanks for the clarification.

**Pull request merged**